host-interaction/file-system/read/read-initialization-file.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: read initialization file
```

Comment:
can we make this name `read .ini file`? i forgot that ".ini" probably stands for "initialization". not a blocker though, if you'd prefer not to change this.

Reply:
Absolutely, I prefer `.ini`